src/ext/snp/mod.rs

```diff
@@ -4,18 +4,18 @@ use std::{fmt::Debug, mem::size_of};
 
 use anyhow::{anyhow, Context, Result};
 
+use super::ExtVerifier;
 use const_oid::db::rfc5912::ECDSA_WITH_SHA_384;
 use const_oid::ObjectIdentifier;
 use der::asn1::UIntBytes;
 use der::{Decodable, Encodable, Sequence};
+use flagset::{flags, FlagSet};
 use pkcs8::AlgorithmIdentifier;
 use sha2::Digest;
 use x509::ext::Extension;
 use x509::{request::CertReqInfo, Certificate};
 use x509::{PkiPath, TbsCertificate};
 
-use super::ExtVerifier;
-
```

Comment:
Don't move this.

Reply:
I think my IDE re-ordered it for some reason.